File: gang/week2/삼각형의 완성조건.java
class Solution {
    public int solution(int[] sides) {

        int minLength = Math.abs(sides[0] - sides[1]) + 1;  //두 변중 더 긴 변을 모르기 때문에 Math.abs로 두 변 길이의 차이에 1을 더함
                                                            //이 값은 나머지 한 변의 최소 길이
        int maxLength = sides[0] + side[1] - 1; //나머지 한 변의 최대 길이

        return maxLength - minLength + 1;   //갯수를 구하는 것이기 때문에 최대 길이에서 최소 길이를 뺀 후 +1
    }
}

File: yeongseong/2week/삼각형의 완성조건.js
function solution(sides) {
    const  min = Math.min(...sides);
    // sides 에서 가장 작은 값을 찾습니다.
    return min * 2 - 1;
    // 가장 작은 값에서 2를 곱하고 1을 빼서 결과릅 받습니다.
}
